File: tests/fixtures/python/safe/realistic_safe_file.py
import os


class UserService:
    def __init__(self, connection):
        self.connection = connection

    def greet(self, name):
        greeting = "Hello, " + name + "!"
        return greeting


MAX_RETRIES = 3
API_ENDPOINT = "https://api.example.com/v1"
DB_PASSWORD = os.getenv("DB_PASSWORD")

File: tests/fixtures/php/safe/realistic_safe_file.php
<?php

class UserService
{
    private $connection;

    public function __construct($connection)
    {
        $this->connection = $connection;
    }

    public function greet($name)
    {
        $greeting = "Hello, " . $name . "!";
        return $greeting;
    }
}

$maxRetries = 3;
$apiEndpoint = "https://api.example.com/v1";
$dbPassword = getenv('DB_PASSWORD');